corehq/toggles.py

```diff
@@ -367,6 +367,14 @@ def toggle_values_by_name(username=None, domain=None):
     [NAMESPACE_DOMAIN, NAMESPACE_USER]
 )
 
+VELLUM_BETA = PredictablyRandomToggle(
+    'vellum_beta',
+    'Use Vellum beta version',
+    TAG_ONE_OFF,
+    [NAMESPACE_DOMAIN],
+    randomness=0.1
```

Comment:
> Just because I just didn't have a method for selecting the "others," decided to pick them randomly.

That's very reasonable.

> so yes, I'll add the domains with the case mgmt feature flags turned on

Rather than maintaining that manually, you could aggregate the flags in code. Something like:
```py
vellum_beta = (toggles.VELLUM_BETA
    or toggles.VELLUM_RICH_TEXT
    or toggles.VELLUM_EXPERIMENTAL_UI)
```
Just a suggestion.